aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -216,12 +217,12 @@ Tensor stft(const Tensor& self, const int64_t frame_length,
   }
   if (hop <= 0) {
     std::ostringstream ss;
-    REPR(ss) << " expected hop > 0, but got hop=" << hop;
+    REPR(ss) << ": expected hop > 0, but got hop=" << hop;
```

Comment:
FYI you can conveniently do this with `AT_CHECK` now (but you would have to rewrite `REPR` macro, so maybe not for this patch)